src/test/java/teammates/test/cases/browsertests/InstructorHomePageUiTest.java

```diff
@@ -350,6 +350,44 @@ private void testPublishUnpublishActions() {
         homePage.waitForTextsForAllStatusMessagesToUserEquals(Const.StatusMessages.FEEDBACK_SESSION_PUBLISHED);
         assertTrue(BackDoor.getFeedbackSession(feedbackSessionPublished.getCourseId(),
                                                feedbackSessionPublished.getFeedbackSessionName()).isPublished());
+
+        ______TS("resend links action: PUBLISHED feedback session");
+        // Test that the resend published links button exists for this published session
+        homePage.verifyResendPublishedLinksButtonExist(feedbackSessionPublished.getCourseId(),
+                                                       feedbackSessionPublished.getFeedbackSessionName());
```

Comment:
Switch your IDE to use 8-space indentation and check all the files in your change set.